blender/arm/logicnode/object/LN_get_visible.py

```diff
@@ -1,12 +1,15 @@
 from arm.logicnode.arm_nodes import *
 
 class GetVisibleNode(ArmLogicTreeNode):
-    """Get visible node"""
+    """Get Visible node"""
     bl_idname = 'LNGetVisibleNode'
     bl_label = 'Get Visible'
+    bl_icon = 'NONE'
 
     def init(self, context):
-        self.add_input('ArmNodeSocketObject', 'Object')
```

Comment:
The same goes for this, https://github.com/armory3d/armory/pull/1860 introduced a `super()` call in `init()` that might be conflicting. Same solution as my other comment.

Reply:
@MoritzBrueckner, okay. I didn't notice this new line there! Thank you :+1:

Reply:
I really don't know what is wrong here. I've added `super(NodeNameNode, self).init(context)` line in all files  but nothing solves the problem. I will  give up until i start using git trough terminal. I will post the files in the forum, this way someone can send the PR. Thank you anyway!

Reply:
Do you have a GUI client for Git like Fork or Github Desktop? They make resolving conflicts a lot easier. But if you want I can create a PR for you, hopefully even with your commits so you get the credit :)

Reply:
All this time i had used the Github site to do everything, but i always end with a lot of commits spammed. It is very limited. I will try to use terminal and test how it works in an own repository until i get familiar to it, but also there is some cool Guis to test, i just wanna one that i can do the things more "cleaner" :+1: 

Feel free to send a PR for me and thank you!